sideboard/config.py

```diff
@@ -9,14 +10,72 @@
 from validate import Validator
 
 
+_defaults_re = re.compile(r'(.+)-defaults(\.\w+)$')
+
+
+def uniquify(x):
+    """
+    Returns an ordered copy of `x` with duplicate items removed.
```

Comment:
I would specify what "ordered" means in this context.  From the implementation I can see that what you mean is "in the order in which the items originally appeared" but a casual reader might assume you meant "in sorted order".